core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/form/content-types-form.component.scss

```diff
@@ -20,3 +20,9 @@
 .content-type__form {
     width: 522px;
 }
+
+.field {
+    .p-field-hint {
+        margin-top: -$spacing-3;
+    }
+}
```

Comment:
Can we make this global?

Reply:
I don't think is needed, because for normal cases, without this styles it looks good. This is just a particular case 

<img width="559" alt="Screenshot 2023-07-11 at 3 40 23 PM" src="https://github.com/dotCMS/core/assets/63567962/028b86ff-7989-4afc-8cc6-00bca37720dd">

Reply:
I made it more specific just in case we add helper text in other fields of this form